### ISSUE TITLE
Feature request: Attenuation aspect on Point Light

### ISSUE THREAD

**Issue by Adolio** [Gamua/Starling-Extension-Dynamic-Lighting#1]
It would be nice to take light attenuation into account for point lights.

Here is a common way to compute the attenuation based on two parameters that LightSource could provide:

`att = 1.0 / (1.0 + linearAttenuationCoeff*dist + quadraticAttenuationCoeff*dist*dist)`

Further info: http://gamedev.stackexchange.com/questions/56897/glsl-light-attenuation-color-and-intensity-formula

Thanks a lot for this extension Daniel :)

**Comment by PrimaryFeather** [comment]
You're right, that's the one part that's still missing — before working on that, I wanted to check if anyone is actually using the extension. :smile:

I'll look into it! In the meantime, anyone who needs this, please "+1" to let me know how to prioritize this feature.

**Comment by lorenzobob0** [comment]
The Dynamic light extension is used and much appreciated. 

This feature would be great!

Thank you for your work!

**Comment by htmiel** [comment]
This would be very useful and a great addition to an already excellent extension!

**Comment by DrNeroCF** [comment]
I hate to admit that I'm only looking into this now, but even in the default state it's still crazy to see this in AIR.  The only real issue I have at the moment is point lights lighting backgrounds at full blast.  Not the most difficult thing to just fade the light's influnce into the background layers, but I'm sure I'm doing it wrong and wouldn't be able to easily have lights in the background, or be able to move lights on the z axis.
